stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/attachments/selected/SelectedFileAttachmentAdapter.kt

```diff
@@ -0,0 +1,60 @@
+package io.getstream.chat.android.ui.attachments.selected
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.getstream.sdk.chat.model.AttachmentMetaData
+import com.getstream.sdk.chat.utils.LlcMigrationUtils
+import com.getstream.sdk.chat.utils.StringUtility
+import io.getstream.chat.android.ui.databinding.StreamItemSelectedAttachmentFileBinding
+
+internal class SelectedFileAttachmentAdapter(
```

Comment:
I'm not a fan of inheritance, but here it could help to avoid code doubling. I see that this adapter and the `SelectedMediaAttachmentAdapter ` have the same logic except of kind view holders. We could create a basic abstract one and two particular inheritors.

Reply:
Done, created a simple abstract RecyclerView adapter